### DISCUSSION TITLE
Can `useHead` accept `Ref`?

### DISCUSSION THREAD

**Discussion by cawa-93** [nuxt/nuxt#17390] [Questions]
In source types `useHead` may accept only `string`
https://github.com/nuxt/framework/blob/ad420238b1e7635e18f6f8cecef868cd2d83e8b3/packages/schema/src/types/meta.ts#L1-L30

But computed `title` seems working

https://stackblitz.com/edit/github-nj7pbx?file=layouts/custom.vue

**Comment by Flanker72** [comment]
Seems that Refs are not supported directly.
However, you may have a look on [Meta Components](https://v3.nuxtjs.org/guide/features/head-management#meta-components): there is a Title which may receive ref through the template.

Here is an example based on your code: https://stackblitz.com/edit/github-nj7pbx-jxfayz?file=layouts%2Fcustom.vue
